ra/registration-authority.go

```diff
@@ -269,6 +271,8 @@ func checkPendingAuthorizationLimit(sa core.StorageGetter, limit *cmd.RateLimitP
 		// here.
 		noKey := ""
 		if count > limit.GetThreshold(noKey, regID) {
+			ra.stats.Inc("RA.RateLimit.PendingAuthorizationsByRegID.Violation", 1, 1.0)
+			ra.log.Info(fmt.Sprintf("Rate limit violation, PendingAuthorizationsByRegID, regID: %d", regID))
```

Comment:
This is super picky, but: how about "Rate limit exceeded" instead of "violation?" The latter sounds too severe and would be distracting in logs to someone unfamiliar with the code. Same with the stat name above.

We should probably also have a stat for `RA.RateLimit.PendingAuthorizationsByRegID.Pass` so we can compare.

Is this a good place to apply the stats scoping stuff you introduced recently?